src/main/java/github/workflow/kata/GOL.java

```diff
@@ -1,15 +1,49 @@
 package github.workflow.kata;
 
-public final class GOL {
-    private GOL() {
-    }
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
 
+public class GOL {
     /**
-     * Starter for this Game Of Life.
+     * This application support loading a world from standard input, or from file with a `-file` parameter.
+     * The format should be:
+     * Generation 1:
+     * 3 3
+     * ...
+     * ***
+     * ...
+     * First line should be the Generation to start with, only 1 is supported.
+     * Second line should be two numbers space separated representing the grid size.
+     * Next lines are the actual grid of the size mentioned above.
      *
-     * @param args the commend line arguments this GOL is started with.
+     * @param args the command line args
      */
-    public static void main(String[] args) {
-        System.out.println("Not working yet.");
+    public static void main(String[] args) throws IOException {
+        // if we have no arguments, we need to read from system.in
+        if (args.length == 0) {
+            System.out.println("Please enter the world definition.");
+            System.out.println("First line should be 'Generation 1'.");
+            System.out.println("Second line should be 3 3.");
+            System.out.println("The next lines are the actual grid, where dead cells are '.' live cells '*'.");
+            System.out.println("Finish the grid with an empty line.");
+            Scanner scanner = new Scanner(System.in);
+            String line = scanner.nextLine();
+            StringBuilder definition = new StringBuilder(line).append("\n");
+            while (line.length() != 0) {
+                line = scanner.nextLine();
+                definition.append(line).append("\n");
+            }
+            new Grid(definition.toString());
+
+        } else if ("-file".equals(args[0])) {
```

Comment:
Missing a -help flag IMO.

Reply:
Yes, definitely. Will add an issue for that